devops/e2e/cloudtest/README.md

```diff
@@ -0,0 +1,22 @@
+# Cloud Test Directory
+
+This directory contains 1ES Managed Image definitions for the distros being exercised by our end-to-end tests.
+
+# General Guidance for onboarding images
+
+Its best to use a pre-existing Azure Marketplace images as they are updated regularly but some EOL distros might require onboarding into a private compute gallery.
+
+## Dependencies
+
+The images defined here are used as VM definitions to be used in Azure infrastructure where we use the VMs as test environments for testing azure-osconfig. The following are requirements for every linux image [Preparing Linux for imaging in Azure](https://learn.microsoft.com/azure/virtual-machines/linux/create-upload-generic) with specific distributions having other set of instructions, see the url provided.
+
+In addition to performing distro specific setup for [Preparing Linux for imaging in Azure](https://learn.microsoft.com/azure/virtual-machines/linux/create-upload-generic) the images must also have the following:
+
+ - [NodeJS v20+](https://github.com/nodejs/node) - Needed by GitHub runners
```

Comment:
What is the expectation for installing these on the image? Can the `AzureInfrastructureHelpers.psm1` assist with this?

Reply:
No, the helpers are mainly for image preparation of the VHD once its already in a good working state

Reply:
The expectation is that the VHD is in working order when it is used